docs/diagnostics/PX1072.md

```diff
@@ -0,0 +1,52 @@
+# PX1072
+This document describes the PX1072 diagnostic.
+
+## Summary
+
+| Code   | Short Description                                                                  | Type                           | Code Fix  | 
+| ------ | ---------------------------------------------------------------------------------- | ------------------------------ | --------- | 
+| PX1072 | BQL queries must be executed within the context of an existing `PXGraph` instance. | Warning (Level 1: Significant) | Available | 
+
+## Diagnostic Description
+The `new PXGraph()` statement should not be used in the execution of BQL queries. Instead, an existing `PXGraph` instance should be used. Because the creation of a `PXGraph` instance is an expensive operation, the instantiation of unnecessary graph instances slows the performance of the application. Also, when you create a new graph, you miss the query results that have already been cached in an existing graph.
```

Comment:
`PXGraph.CreateInstance` should also be mentioned

Reply:
fixed